web/src/components/pages/brandsPage/BrandsPage.tsx

```diff
@@ -0,0 +1,18 @@
+import { useContext } from 'react';
+import { PageResourceContext } from '../../../contexts';
+import { BrandItem } from './BrandItem';
+import './style.css';
+
+export const BrandsPage = () => {
+    const {
+        pageResource: { brands },
+    } = useContext(PageResourceContext);
+
+    return (
+        <div className="brandItems__container">

```

Comment:
just className="brands" or brands__container

items are below and this div is some sort of wrapper (parent element)